keybot/platform_linux.go

```diff
@@ -30,6 +30,10 @@ func buildAndroidCommand() slackbot.ExecCommand {
 	return slackbot.NewExecCommand("bash", []string{"-c", "echo not implemented; false"}, true, "Perform an alpha build")
 }
 
+func buildIOSCommand() slackbot.ExecCommand {
```

Comment:
we should refactor so we don't have all these empty commands in the future, but ok for now